internal/services/web/app_service_resource_test.go

```diff
@@ -1294,6 +1294,43 @@ func TestAccAppService_windowsJava11Java(t *testing.T) {
 	})
 }
 
+
+func TestAccAppService_windowsJava17Java(t *testing.T) {
```

Comment:
The new value you are trying to add wasn't added to this resource so these new tests are failing. I believe you want to add these tests to `windows_app_service_resource_test.go` or to add the new value to this resources schema

Reply:
Sorry about that. The change attempt was done by searching for "java_version" and not paying to much attention to what was changes. I was going to try to run the tests, but could not find any description of what ARM_CLIENT_ID and ARM_CLIENT_SECRET, and gave up.

I have redone the changes now, following the pointers from (windows|linux)_web_app_resource.go and (windows|linux)_function_app_resource.go